src/renderer/dialogs/MokRestartDialog.tsx

```diff
@@ -0,0 +1,135 @@
+import Channels from 'constants/Channels'
+import { cloneCluster } from 'models/Cluster'
+import { ShellResponse } from 'models/ShellResponse'
+import { enqueueSnackbar } from 'notistack'
+import { SettingsService, useSettingsState } from 'renderer/services/SettingsService'
+
+import { Avatar, Box, Button, Dialog, DialogActions, DialogContent, DialogTitle, Typography } from '@mui/material'
+
+import logoMinikube from '../../../assets/icons/minikube.png'
+
+interface Props {
+  onClose: () => void
+}
+
+const MokRestartDialog = ({ onClose }: Props) => {
+  const settingsState = useSettingsState()
+  const selectedCluster = settingsState.value.mokCluster
+
+  const onRestart = async () => {
+    try {
+      const clonedCluster = cloneCluster(selectedCluster!)
+
+      const password = await SettingsService.getDecryptedSudoPassword()
+
+      const command = `echo '${password}' | sudo -S systemctl reboot`
```

Comment:
I think this command can be moved to Base Clster commands and then `sudo` can be replaced as suggested above.